### DISCUSSION TITLE
Re-send verification email

### DISCUSSION THREAD

**Discussion by kencx** [firefly-iii/firefly-iii#5811] [Firefly III] [answered]
I managed to stupidly lock myself out by changing my admin account email. Firefly now prompts me to verify my new email before I can login. 

However, it was sent before I configured the email environment variables so I have lost the verification email. I have configured them now and am able to receive emails through the Docker log.

Is there an alternative way to re-send the verification email?

Thanks a lot.

**Comment by JC5** [comment, marked as answer]
No, but since you have access to the database you can overrule the change yourself. 

In table `users`, change `blocked` to 0 and set `blocked_code` to NULL. In the `preferences` table remove the preferences that refer to your old/new email address.

That should validate the change as well.

**Reply by kencx** [reply]
That worked, thank you for you help! 

May I also suggest adding a "resend verification email" button in the verify email prompt?

**Reply by JC5** [reply]
It doesn't happen all too often, so that may not be something you'll see anytime soon.

**Reply by b3nd1k** [reply]
Hey! I've ran into the same issue, but have no idea how to correcy change the database. I'm using Adminer to access my MariaDB database, could I get some guideance? 

This is the user tab
<img width="886" alt="Screenshot 2022-04-03 at 15 34 27" src="https://user-images.githubusercontent.com/96381857/161430624-467688b4-e6cf-492d-9cc8-f40c159b3789.png">

This is the preferences tab. 
<img width="860" alt="Screenshot 2022-04-03 at 15 35 18" src="https://user-images.githubusercontent.com/96381857/161430655-7caa875e-00bd-40fc-8c2a-c9113799b3c8.png">

What should I do?

**Reply by niklas2810** [reply]
Hey @b3nd1k , the first screenshot looks like  an overview of the table's columns, not the data inside of the columns 😄 Maybe you have another view where you can change the data itself and not the table?

If you find it, go to the table `users`, set `blocked_code` to NULL (this might be a checkbox in your tool if you cant select it) and `blocked` to `0` :wink:

After a search I found this guide: https://kinsta.com/blog/adminer/#insert-new-records-and-update-existing-ones I have never worked with your tool before, sorry about that, but it seems like there should be an "Edit" dialog somewhere!
